Plane.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace task_Dev_5
+{
+    /// <summary>
+    /// class Plane and its properties are described
+    /// </summary>
+    class Plane : IFlyable
+    {
+        Point point;
+
+        /// <summary>
+        /// stores information about the speed of flight of the plane
+        /// </summary>
+        double planeFlightSpeed;
+
+        /// <summary>
+        /// flight time of the plane
+        /// </summary>
+        double flightTime;
+
+        /// <summary>
+        ///Variable storing distance of plane flight
+        /// </summary>
+        double distance;

```

Comment:
it is not needed to add summary for all class fields. Use  valid names for variables. It will be enough